poc/policy/multi_hot_histogram_with_client_dp.py

```diff
@@ -0,0 +1,67 @@
+"""
+A policy that uses only Client-DP to achieve central (eps, delta)-DP.
+
+Each Client adds binary randomized response noise with pure `eps0`-DP.
+With `min_batch_size` number of honest Clients, the final aggregate
```

Comment:
Why talk about the min batch size and not the concrete batch size? (Clarify in the comment.)